### DISCUSSION TITLE
Rescheduling a recurring task without removing all future recurring tasks

### DISCUSSION THREAD

**Discussion by AlphabetsAlphabets** [davish/obsidian-full-calendar#536] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I would like to re-schedule or cancel a recurring task without it affecting future recurring tasks. When deleting a task, it would be nice to have the UI like Proton Calendar. 

![image](https://github.com/davish/obsidian-full-calendar/assets/44822875/d41164fc-bc37-4e1d-a90e-f51a498f0e22)

### ✔️ Solution

For example, I have a recurring task every Thursday at 8:30 to 10:30. However, for one reason or another, that class is cancelled for the next week. Like like to delete a task **only for the next week**. And when I look at the Calendar for the next week, that task is still there.

Currently, when I delete a recurring task, it deletes all instances of that task from the Calendar and I'll have to add it back in.

### ❓ Alternatives

I would add a new event called "SNA class cancelled." Then schedule other things on top of it.

**Comment by Cerabbite** [comment]
Maybe add a 3rd option for removing only events after the one you are trying to delete.

So you have

- Delete this event (deletes this and only this event)
- Delete future events (deletes this and all events scheduled after the one you are trying to delete)
- Delete all events (deletes all events regardless of when it is scheduled)

**Comment by DigitalDicast** [comment]
This would be awesome

**Comment by Anonyomous-Anonyomous** [comment]
The plugin is borderline unusable for daily schedules if this option isn't added

**Comment by YouFoundJK** [comment]
This should now be possible in my fork [here](https://github.com/YouFoundJK/plugin-full-calendar/releases), if anyone is interested.
